fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/utils/ColumnValidator.java

```diff
@@ -91,6 +92,9 @@ private void validateColumn(Map<String, Set<String>> tableColumnMap) {
 
     public void validateSqlInjection(String schema, String... conditions) {
         for (String condition : conditions) {
+            if (!StringUtils.isNotBlank(condition)) {
```

Comment:
Why the "double negation" and not just if isBlank() ?

Reply:
Opps, updating.